packages/frontEnd/src/views/categoryDetails/ProductList.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import { ProductCard } from "views/home/recommended/ProductCard";
+import AdService from "../../services/AdService";
+import { Pagination } from "./pagination/Pagination";
+
+interface ProductProps {
+  productImages: Array<string>;
+  productTitle: string;
+  price: number;
+  adCategory: string;
+  city: string;
+}
+
+export const ProductList = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [products, setProducts] = useState<null | ProductProps[]>(null);
```

Comment:
if you render the filter page by default, the data should load. If you submit a query and the query matches a filter, data will change.